corehq/apps/callcenter/tests/test_utils.py

```diff
@@ -194,44 +199,47 @@ def test_sync_usercase_custom_user_data_on_update(self):
             'completed_training': 'yes',
         }
         sync_usercase(self.user)
-        case = get_case_by_domain_hq_user_id(TEST_DOMAIN, self.user._id, USERCASE_TYPE)
-        self.assertEquals(case.completed_training, 'yes')
+        case = CaseAccessors(TEST_DOMAIN).get_case_by_domain_hq_user_id(self.user._id, USERCASE_TYPE)
+        self.assertEquals(case.dynamic_case_properties()['completed_training'], 'yes')
 
+    # @run_with_all_backends
```

Comment:
This test and the next aren't enable because we don't have the ability to easily find which form was used to close the case for SQL cases: https://github.com/dimagi/commcare-hq/blob/0447042cca2c1e06e060fe1583a79fbd1d25dcd3/corehq/apps/callcenter/utils.py#L55-L55